examples/python/mapping/full_pipeline.py

```diff
@@ -0,0 +1,122 @@
+import os
+import csv
+from PIL import Image as PilImage
+import numpy as np
+import pywavemap as wave
+
+# Parameters
+home_dir = os.path.expanduser('~')
+measurement_dir = os.path.join(home_dir,
+                               "data/panoptic_mapping/flat_dataset/run2")
+output_map_path = os.path.join(home_dir, "your_map.wvmp")
```

Comment:
If you wanted you could also use `argparse` to get those as command line params from the user.